src/Avalonia.Controls/TopLevel.cs

```diff
@@ -185,7 +185,7 @@ IStyleHost IStyleHost.StylingParent
         }
 
         /// <inheritdoc/>
-        IRenderTarget IRenderRoot.CreateRenderTarget()
+        public virtual IRenderTarget CreateRenderTarget()
```

Comment:
Could we add an explicit interface implementation that calls the protected method?